### PR TITLE
Slurpit Support

### PR DESCRIPTION
**Pull request by lpconsulting321**
Implements: #600 

Added support for slurpit to sync locations, device types, devices, interfaces, inventory items, prefixes, vlans, vrf's and IP's. 

Let me know if there are any issues.

### CODE REVIEW

**Review by jdrew82 — APPROVED** [review]
LGTM! Good work and awesome to see an async example with the contrib pattern!

**Comment by jdrew82** [comment]
@lpconsulting321 One thing I just realized I forgot about. Can you update the CODEOWNERS in the github folder with who is responsible for the integration? Would it be just you or are there others?

**Comment by lpconsulting321** [comment]
@jdrew82 Yeah. I wanted to use the built in Nautobot adapter as much as possible. Possible improvements for that in future would be to allow the ability allow `get_queryset` to pull based on tags/cf's I overcome this by just overiding the functions. Other than that it was perfect.

**Comment by lpconsulting321** [comment]
@jdrew82 

`/nautobot_ssot/integrations/slurpit/ @lpconsulting321 @pietos @nautobot/plugin-ssot` is this okay or shall i remove the last user?

**Comment by jdrew82** [comment]
> @jdrew82
> 
> `/nautobot_ssot/integrations/slurpit/ @lpconsulting321 @pietos @nautobot/plugin-ssot` is this okay or shall i remove the last user?

You can leave the plugin-ssot group as a fallback just in case it's something we can easily address.

**Comment by lpconsulting321** [comment]
Do you know what the issue might be on the pylint?

**Review by jdrew82 — APPROVED** [review]
LGTM!

**Comment by lpconsulting321** [comment]
@jdrew82 Looks like unit tests are failing for non slurpit integrations. What do you do in these situations?

File: development/development.env [hunk 1]
```diff
@@ -110,3 +110,6 @@ IPFABRIC_SSL_VERIFY="True"
 IPFABRIC_TIMEOUT=15
 
 NAUTOBOT_SSOT_ENABLE_ITENTIAL="True"
+
+NAUTOBOT_SSOT_ENABLE_SLURPIT="True"
```
Comment (jdrew82) [inline development/development.env:114]: This should be False.

Comment (jdrew82) [reply development/development.env:114]: @lpconsulting321 That is most likely being caused due to your integration being enabled. Making this change should resolve the issue.

Comment (lpconsulting321) [reply development/development.env:114]: <3

**Comment by lpconsulting321** [comment]
Ahh 3.8. The Slurpit SDK does not support 3.8 at the moment. We will make sure our SDK supports 3.8 and get this passing.

**Comment by jdrew82** [comment]
@lpconsulting321 Nautobot 2.4 will be removing support for Python 3.8 so you might want to wait for that to be released and we can set that as minimum for this integration?

**Comment by jdrew82** [comment]
Oh, also, before I forget, please make sure you update the docs in the README to include you integration in the list and the acknowledgements.

**Pull request closed without merging**